source/plugin/placeholders/creatingtokens.rst

```diff
@@ -30,7 +34,7 @@ There are two ways you can create a ``PlaceholderParser``:
 * Directly implement the interface.
 
 .. note::
-  ``PlaceholderParsers`` are :doc:`Catalog Types<../data/catalog-types>`. If you implement the interface directly,
+  ``PlaceholderParsers`` are Catalog Types (FIXME). If you implement the interface directly,
```

Comment:
FIXME. Sweet, a placeholder in the placeholder docs. Did you intend to flesh this out?

Reply:
Catalog Types have been removed or at least renamed so I removed the no longer working link and replaced it with Fixme. This section has to be updated eventually but not neccesarily in this PR.

Reply:
OK, FIXME is probably as good as TBA or "Coming Soon".

Reply:
CatalogTypes are not a thing any more. I'll just remove that part for now.